Moovies/Moovies/DescriptionModule/MoovieDescriptionTableViewController.swift

```diff
@@ -0,0 +1,88 @@
+// MoovieDescriptionTableViewController.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+class MoovieDescriptionTableViewController: UITableViewController {
+    // MARK: Enum
+
+    enum Cells {
+        case poster
+        case overview
+    }
+
+    // MARK: Internal Properties
+
+    var movieID = Int()
+
+    // MARK: Private Properties
+
+    private var details: Description?
```

Comment:
```suggestion
    private var details: Description?
```
это же модель ее в перезтер